app/views/admin/orders/show.html.erb

```diff
@@ -0,0 +1,74 @@
+    <h4> Shipping City: <%= @order.info.city %> </h4>
+    <h4> Shipping State: <%= @order.info.state %> </h4>
+    <h4> Shipping Zipcode: <%= @order.info.postal_code %> </h4>
+    <h4> Shipping Country: <%= @order.info.country %> </h4>
+    <h4> Order Status: <%= @order.status %></h4>
+
+
+  </div>
+
+  <table class="table table-hover table-condensed">
+  	<thead>
+  		<tr>
+        <th class="col-sm-6"><%= "Item" %></th>
+        <th class="col-sm-2"><%= "Price" %></th>
+        <th class="col-sm-2"><%= "Quantity" %></th>
+  			<th class="col-sm-2" class="text-center"><%= "Subtotal" %></th>
+  		</tr>
+  	</thead>
+  	<tbody>
+      <% @items.each do |order_item| %>
+  		<tr>
+  			<td data-th="Product">
+  				<div class="row">
+  					<div class="col-sm-2 hidden-xs">
+              <img><%= image_tag order_item.first.image, class: "img-responsive" %></img>
+            </div>
+  					<div class="col-sm-10">
+  						<h3 class="nomargin"><%= link_to "#{order_item.first.title}", item_path(order_item.first.id) %></h3>
+              <p><%= order_item.first.description %></p>
+  					</div>
+  				</div>
+  			</td>
+  			<td data-th="Price"><%="$#{order_item.first.price}" %></td>
+        <td data-th="Quantity"><%= order_item.count %></td>
+        <td data-th="Subtotal"><%= "$#{subtotal(order_item.first.price, order_item.count)}" %></td>
+        <% end %>
+  		</tr>
+  	</tbody>
+    <td class="col-lg-2 col-lg-pull-5"><strong>Order Total: <%= "$#{order_total(@items)}" %></strong></td>
+  	<tfoot>
+  		<tr class="visible-xs">
+  		</tr>
+  		<tr>
+        <% if @order.status == "Ordered" %>
+          <td><h2> Edit Order Status <h2></td>
+          <td><a><%= button_to "Cancel", admin_order_path, action: :update_cancel, method: :post, class: "btn btn-danger btn-large", name: "commit", value: "Canceled",  :html => {:class => "form-horizontal center"}, id: "Canceled" %></a></td>
+          <td><a><%= button_to "Mark as Paid", admin_order_path, action: :update_mark_as_paid, method: :post, class: "btn btn-primary btn-large", name: "commit", value: "Paid", :html => {:class => "form-horizontal center"}, id: "Paid" %></a></td>
+        <% elsif @order.status == "Paid" %>
+          <td><h2> Edit Order Status <h2></td>
+          <td><a><%= button_to "Cancel", admin_order_path, action: :update, method: :post, class: "btn btn-danger btn-large", name: "commit", value: "Canceled", :html => {:class => "form-horizontal center"}, id: "Canceled" %></a></td>
+          <td><a><%= button_to "Completed", admin_order_path, action: :update, method: :post, class: "btn btn-primary btn-large", name: "commit", value: "Completed", :html => {:class => "form-horizontal center"}, id: "Completed" %></a></td>
+        <% elsif @order.status == "Canceled" %>
+          <td><h2>Order has been Canceled <h2></td>
+        <% else @order.status == "Completed" %>
+          <td><h2>Order has been Completed <h2></td>
+        <% end %>
```

Comment:
A lot going on here and it's too much logic for the view. Think of ways to clean this up.. maybe a helper method like `ordered?` or `paid?`. Partials could also be great here to and could help eliminate some ifs and DRY up the code.